src/components/RadioChoiceField/RadioChoiceField.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from "react"
+import { useCallback, useMemo, useId } from "react"
+import FormControl from "@mui/material/FormControl"
+import FormControlLabel from "@mui/material/FormControlLabel"
+import Radio from "@mui/material/Radio"
```

Comment:
MIT Learn has a custom `Radio` component. In practice, it isn't used anywhere; RadioChoiceField is what we use, and it is using MUI's.